.github/workflows/k8s-content-pr-test.yaml

```diff
@@ -0,0 +1,58 @@
+---
+name: Gate / Kubernetes Test Content Parsing
+
+on:
+  pull_request:
+    types:
+    - opened
+    - reopened
+    - synchronize
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.number || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  datastream-parsing:
+    name: XCCDF Datastream parsing
+    runs-on: ubuntu-latest
+    steps:
+      - name: Copy XCCDF files from existing content image
+        uses: nick-fields/retry@v3
+        with:
+          timeout_minutes: 15
+          max_attempts: 3
+          retry_wait_seconds: 120
+          retry_on: error
+          command: |
+            mkdir -p content
+            docker pull ghcr.io/complianceascode/k8scontent:${{ github.event.number }}
+            docker run --rm -v $PWD/content:/content:z ghcr.io/complianceascode/k8scontent:${{ github.event.number }} bash -c "cp *.xml /content"
+      - name: Clone compliance operator repository
+        run: |
+          git clone https://github.com/ComplianceAsCode/compliance-operator.git
+      - uses: actions/setup-go@v5
+        with:
+          go-version: '>=1.19.0'
+      - run: go version
+      - name: Install ginkgo
+        run: |
+          go install github.com/onsi/ginkgo/ginkgo@latest
```

Comment:
This version of ginkgo may be different than what's tracked as a dependency of the Compliance Operator.

What if we invoked this through something like:

```console
git clone https://github.com/ComplianceAsCode/compliance-operator.git
cd compliance-operator
make test-datastreams
```

Then `make test-datastreams` could look for all data stream files in a particular test directory, and run the tests. The final step here would check if the `make` target is successful or not through return codes.

This would push more of the implementation details of the test, and how it's invoked into the compliance-operator code base, while leaving a relatively opaque interface (`make test-datastreams`) for the content repository to call.